.github/workflows/test.yml

```diff
@@ -0,0 +1,204 @@
+name: Code Build and Tests
```

Comment:
This means unit and integration tests will run sequentially? Can we run them in paralle?